<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_sentinel</class>
 <widget class="QWidget" name="import_sentinel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导入哨兵一号数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>导入</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5,2">
         <item>
          <widget class="QLabel" name="label_manifest_file">
           <property name="text">
            <string>哨兵图像文件（.safe）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_manifest_file"/>
         </item>
         <item>
          <widget class="QPushButton" name="browse_Button">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,5,2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>精轨文件（可空缺）   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_POD"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_POD">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>子带选择（subswath）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComboBox_subswath"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>极化方式选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComboBox_polarization"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>目标工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_dst_project"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dst_node"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>目标文件名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineEdit_dst_filename"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>批量导入</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="8,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListWidget" name="listWidget"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_add">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_remove">
             <property name="text">
              <string>移除</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>子带选择（subswath）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComboBox_subswath_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>极化方式选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComboBox_polarization_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>目标工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComboBox_dst_project_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>目标节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dst_node_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="5,5">
         <item>
          <widget class="QProgressBar" name="progressBar_2">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox_2">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
